client/src/components/Banner/Banner.css

```diff
@@ -0,0 +1,28 @@
+.Banner{
+  padding: 0;
+  margin: 0 auto;
+  width: 100%;
+  width: 100vw;
+  height: 231px;
```

Comment:
You probably want the height to be determined by the content inside the banner, rather than have a set height